### ISSUE TITLE
ColorCop acting like crazy in monitor UHD 4K

### ISSUE THREAD

**Issue by Z3n3K** [ColorCop/ColorCop#18]
The cursor selects the wrong area of the screen.

**Comment by darkworks** [comment]
looks like project is dead

**Comment by Z3n3K** [comment]
> looks like project is dead

Yes men sadly. I am using this right now http://annystudio.com/software/colorpicker/ It is working well in a windows 10 with a UHD 4k monitor

**Comment by joshlang** [comment]
A solution that worked for me:
- Right click on Colorcop.exe
- Click Properties -> Compatibility
- Click "Change high DPI settings"
- Click both checkboxes

Then it works as expected.

#17 #16 #13 #12 #9

**Comment by dodedodo** [comment]
I can confirm this fix works. Thanks joshlang! No more hunting around with the picker tool🤡

@j4y Could you look into this one more time? Maybe you can enable these settings by default in the app.

**Comment by David263** [comment]
I found this fix too, and it works for the other screen color/measuring tools also. Once again, we have Microsoft to thank for requiring us to specify when backwards compatibility is needed, instead of building in compatibility automatically when needed.

**Comment by j4y** [comment]
It looks like there is a way to fix this:

https://learn.microsoft.com/en-us/windows/win32/hidpi/high-dpi-desktop-application-development-on-windows

Microsoft was nice enough to put out that document.

**Comment by j4y** [comment]
Thanks everyone for your patience while we tracked this down.

The fix is now live in #262, and the app embeds the updated manifest as of commit b2ca976.

This should address the DPI/theming issues reported earlier.
